README.md

```diff
@@ -6,49 +6,28 @@
 
 ## Docker Environment
 
-To use the docker environment clone this repo inside a folder structure like this:
-
-```
-some_directory <- Name is not important
-  |- package <- This repo code inside that folder
-  |- src <- An empty folder to be used in the automatic installation process
-```
-
-And copy this repo's `docker-compose.yml` & `build.sh` & `.env` files to the root of the base directory (`some_directory` in the example).
-
 > **NOTE**: If you are in Windows or WSL, probably you'll have to fix CONTROL-M (^M) carriage return characters in `build.sh` file. Run this command to get rid of this characters:
 
 ```
 dos2unix build.sh
 ```
 
-So you'll have:
-
-```
-some_directory <- Name is not important
-  |- package <- This repo code inside that folder
-  |- src <- An empty folder to be used in the automatic installation process
-  |- docker-compose.yml
-  |- build.sh
-  |- .env
-```
-
-Edit the copy of `.env` and set your tokens in the `COMPOSER_AUTH` environment variable. You can also set the version of Magento you wish to install.
+Rename the `.env.example` to `.env` and set your tokens provided by magento at the [admin panel](https://commercemarketplace.adobe.com/customer/accessKeys) in the `COMPOSER_AUTH` environment variable. You can also set the version of Magento you wish to install.
 
 Then run the environment by executing:
 
 ```
-$ docker-compose up
+$ docker-compose up --profile setup
```

Comment:
I've just tested this and it does not work this way, but it requires a very small change to fix it:
```suggestion
$ docker-compose --profile setup up
```
The `up` has to be at the end according to the docs: https://docs.docker.com/compose/profiles/#start-specific-profiles

Reply:
si, tienes razon... :+1: